setup_docker_ebs.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# Provision and mount EBS volume to /var/lib/docker
+sudo mkfs -t xfs -n ftype=1 $DOCKER_EBS_NAME
+sudo mkdir -vp /var/lib/docker
+sudo mount $DOCKER_EBS_NAME /var/lib/docker
+sudo bash -c "echo -e '$DOCKER_EBS_NAME\t/var/lib/docker\txfs\tdefaults,nofail\t0\t2' >> /etc/fstab"
```

Comment:
Let's add some systemd constraints here so the mount for sure happens before docker starts.
```suggestion
sudo bash -c "echo -e '$DOCKER_EBS_NAME\t/var/lib/docker\txfs\tdefaults,nofail,x-systemd.before=docker.service\t0\t2' >> /etc/fstab"
```